### PR TITLE
Add Glonass support to signal.c/h

### PR DESCRIPTION
**Pull request by pmiettinen**

### CODE REVIEW

**Comment by ljbade** [comment]
LGTM. Coveralls failure is strange, only new executable code is in the test.

**Comment by pmiettinen** [comment]
@ljbade I made addition to unit test to bring up the signal.c coverage to 100.

**Comment by ljbade** [comment]
Excellent, I think it's ready to merge

**Comment by pmiettinen** [comment]
@ljbade Who does the merging? I don't have rights.

**Comment by ljbade** [comment]
I can

File: include/libswiftnav/signal.h [hunk 1]
```diff
@@ -21,26 +21,34 @@
...
+#define NUM_SIGNALS_GPS   (NUM_SIGNALS_GPS_L1CA + NUM_SIGNALS_GPS_L2CM)
+#define NUM_SIGNALS_SBAS  (NUM_SIGNALS_SBAS_L1CA)
+#define NUM_SIGNALS_GLO   (NUM_SIGNALS_GLO_L1CA + NUM_SIGNALS_GLO_L2CA)
+#define NUM_SIGNALS       (NUM_SIGNALS_GPS + NUM_SIGNALS_SBAS +\
+                           NUM_SIGNALS_GLO)
 
-#define GPS_FIRST_PRN 1
-#define SBAS_FIRST_PRN 120
+
+#define GPS_FIRST_PRN   1
```
Comment (jacobmcnamee) [inline include/libswiftnav/signal.h]: Might want to rename these to *_FIRST_SAT eventually.

**Pull request merged**